app/export.go

```diff
@@ -62,6 +65,62 @@ var exportablePreferences = map[ComparablePreference]string{{
 }: "EmailInterval",
 }
 
+func (a *App) GetID() string {
+	return a.ID
+}
+func (a *App) SetID(id string) {
+	a.ID = id
+}
+
+func exportLogFile(attachment AttachmentImportData, exportFilename string) (*LogFormat, *model.AppError) {
```

Comment:
What's the use of `exportFilename` here?

Reply:
Removed